test/integration/commands/excel2json/test_new_lists.py

```diff
@@ -34,9 +36,87 @@ def test_second_list(expected_lists: list[dict[str, Any]], result_lists_correct:
     assert result_lists_correct[1] == expected_lists[1]
 
 
-def test_thrid_list(expected_lists: list[dict[str, Any]], result_lists_correct: list[dict[str, Any]]) -> None:
+def test_third_list(expected_lists: list[dict[str, Any]], result_lists_correct: list[dict[str, Any]]) -> None:
     assert result_lists_correct[2] == expected_lists[2]
 
 
+def test_duplicate_list_id() -> None:
+    expected = regex.escape(
+        "\nThe excel file(s) used to create the list section have the following problem(s):"
+        "\n\n---------------------------------------\n\n"
```

Comment:
I think this separator shouldn't exist here.

Reply:
I disagree, this formatting visually separates multiple problems better. This is not the case here, because we would have to be really careful about the sorting everywhere to make a test that always passes. Therefore I do not want to do that. But I think it will look better when there are multiple problems.

Reply:
I find it useful if several problems are separated, but not if there is a separator at the beginning.